TestAnimals/UnitTest1.cs

```diff
@@ -0,0 +1,45 @@
+using AnimalsTest;
+
+namespace TestAnimals
+{
+    public class UnitTest1
```

Comment:
Avoid default naming here
```suggestion
    public class BasicTests
```

```suggestion
    public class MiscTests
```